cartography/intel/aws/redshift.py

```diff
@@ -0,0 +1,118 @@
+import logging
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_redshift_cluster_data(boto3_session, region):
+    client = boto3_session.client('redshift', region_name=region)
+    paginator = client.get_paginator('describe_clusters')
+    clusters = []
+    for page in paginator.paginate():
+        clusters.extend(page['Clusters'])
+    return clusters
+
+
+def _make_redshift_cluster_arn(region, aws_account_id, cluster_identifier):
+    return f'arn:aws:redshift:{region}:{aws_account_id}:cluster:{cluster_identifier}'
+
+
+@timeit
+def transform_redshift_cluster_data(clusters, region, current_aws_account_id):
+    for cluster in clusters:
+        # Cluster ARN format https://docs.aws.amazon.com/redshift/latest/mgmt/redshift-iam-access-control-overview.html
+        cluster['arn'] = _make_redshift_cluster_arn(region, current_aws_account_id, cluster["ClusterIdentifier"])
+
+
+@timeit
+def load_redshift_cluster_data(neo4j_session, clusters, region, current_aws_account_id, aws_update_tag):
+    ingest_cluster = """
+    MERGE (cluster:RedshiftCluster{id: {Arn}})
+    ON CREATE SET cluster.firstseen = timestamp()
+    SET cluster.availability_zone = {AZ},
+    cluster.cluster_create_time = {ClusterCreateTime},
+    cluster.cluster_identifier = {ClusterIdentifier},
+    cluster.cluster_revision_number = {ClusterRevisionNumber},
+    cluster.db_name = {DBName},
+    cluster.encrypted = {Encrypted},
+    cluster.cluster_status = {ClusterStatus},
+    cluster.endpoint_address = {EndpointAddress},
+    cluster.endpoint_port = {EndpointPort},
+    cluster.master_username = {MasterUsername},
+    cluster.node_type = {NodeType},
+    cluster.number_of_nodes = {NumberOfNodes},
+    cluster.publicly_accessible = {PubliclyAccessible},
+    cluster.vpc_id = {VpcId},
+    cluster.lastupdated = {aws_update_tag},
+    cluster.region = {Region}
+    WITH cluster
+    MATCH (aa:AWSAccount{id: {AWS_ACCOUNT_ID}})
+    MERGE (aa)-[r:RESOURCE]->(cluster)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {aws_update_tag}
+    """
+    for cluster in clusters:
+        cluster_create_time = str(cluster['ClusterCreateTime']) if 'ClusterCreateTime' in cluster else None
```

Comment:
Can be condensed to:
cluster_create_time = str(cluster.get('ClusterCreateTime'))

Reply:
Oh, ignore.  I see it now.  We want to store `None`